packages/atlas/src/components/_navigation/SidenavStudio/SidenavStudio.tsx

```diff
@@ -39,12 +39,16 @@ const studioNavbarItems: NavItemType[] = [
     expandedName: 'My uploads',
     to: absoluteRoutes.studio.uploads(),
   },
-  {
-    icon: <SvgSidebarPayments />,
-    name: 'Payments',
-    expandedName: 'My payments',
-    to: absoluteRoutes.studio.payments(),
-  },
+  ...(atlasConfig.features.ypp.googleConsoleClientId
+    ? []
+    : [
+        {
+          icon: <SvgSidebarPayments />,
+          name: 'Payments',
+          expandedName: 'My payments',
+          to: absoluteRoutes.studio.payments(),
+        },
+      ]),
```

Comment:
I don't think we should just hide the payments from the navigation. On gleev we still have payments in navigation, although it's not working - we have a placeholder screen there.
![image](https://user-images.githubusercontent.com/51168865/216537217-ff7d41af-d244-4cb2-be4e-1041b51381c6.png)

Removing the payments nav item together with the placeholder screen could suggest to the users that we abandoned the payments feature when we release YPP.